apps/users/forms.py

```diff
@@ -0,0 +1,86 @@
+from django import forms
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
+import re
+from decouple import config
+
+
+class ChangePasswordForm(forms.Form):
+    password = forms.CharField(
+        widget=forms.PasswordInput(attrs={'class': 'form-control'}),
+        label="New Password"
+    )
+    confirm_password = forms.CharField(
+        widget=forms.PasswordInput(attrs={'class': 'form-control'}),
+        label="Confirm Password"
+    )
+
+    def clean(self):
+        cleaned_data = super().clean()
+        password = cleaned_data.get("password")
+        confirm_password = cleaned_data.get("confirm_password")
+        if password and confirm_password and password != confirm_password:
+            raise forms.ValidationError("Passwords do not match.")
+        return cleaned_data
+
+class LoginForm(forms.Form):
+    username = forms.CharField(max_length=config("USERNAME_MAX_LENGTHT"), required=True)
+    password = forms.CharField(max_length=config("PASSWORD_MAX_LENGTHT"), required=True, widget=forms.PasswordInput)
+
+    def clean_username(self):
+        username = self.cleaned_data['username']
+        if not re.match(r'^[\w]+$', username):
+            raise ValidationError("Login need to involve only letters, digits and underscores.")
+        return username
+
+class RegisterForm(forms.Form):
+    username = forms.CharField(max_length=config("USERNAME_MAX_LENGTHT"), required=True)
+    email = forms.EmailField(required=True)
+    password = forms.CharField(max_length=config("PASSWORD_MAX_LENGTHT"), required=True, widget=forms.PasswordInput)
+
+    def clean_username(self):
+        username = self.cleaned_data['username']
+        if not re.match(r'^[\w]+$', username):
+            raise ValidationError("Login need to involve only letters, digits and underscores.")
+        if User.objects.filter(username=username, is_active=True).exists():
+            raise ValidationError("Login is used by another user.")
+        return username
+
+    def clean_email(self):
+        email = self.cleaned_data['email']
+        if User.objects.filter(email=email, is_active=True).exists():
+            raise ValidationError("This email is already registered.")
+        if len(email) > 50:
```

Comment:
I don't agree that this is good validation. What you should do is check if the input text is actually an email. Look for libraries in Python for this.